packages/components/src/components/map-marker/map-marker.stories.ts

```diff
@@ -6,8 +6,12 @@ import { withActions } from '@storybook/addon-actions/decorator';
 const { argTypes, parameters } = storybookDefaults('sd-map-marker');
 const { generateTemplate } = storybookTemplate('sd-map-marker');
 
+/**
+ * Used to show a location or a cluster of locations on a map.
+ */
 export default {
   title: 'Components/sd-map-marker',
+  tags: ['!dev'],
   component: 'sd-map-marker',
   parameters: {
     ...parameters
```

Comment:
figma link?

Reply:
@coraliefeil @MartaPintoTeixeira can you check if this is the correct figma link? It's the only I could find related to this component: https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=2544-17460&node-type=FRAME&m=dev

Reply:
Clarified in teams call